examples/models/llama/runner/eager.py

```diff
@@ -74,6 +75,12 @@ def build_args_parser() -> argparse.ArgumentParser:
         help="Have multi-turn chat with the model",
     )
 
+    parser.add_argument(
+        "--tokenizer_config_path",
+        type=str,
+        default=None,
+    )
```

Comment:
Please add `help` to clarify how to use this argument

Reply:
Also this should be mutually exclusive with tokenizer_path? We should make sure only one is passed in.

Reply:
Should be just metadata like [this](https://huggingface.co/Qwen/Qwen2.5-1.5B/blob/main/tokenizer_config.json), e.g. it tells you what the eos_token is, which isn't info that is in the `tokenizer.json` itself